spec/index.bs

```diff
@@ -4687,9 +4700,9 @@ the behavior the same as when the format is unknown to the implementation.
 
 See [[#texture-format-caps]] for information about which {{GPUTextureFormat}}s require features.
 
-<div algorithm>
-    <dfn abstract-op>Validate texture format required features</dfn> of a {{GPUTextureFormat}}
-    |format| with logical [=device=] |device| by running the following steps:
+<div algorithm class=validusage data-timeline=content>
+    <dfn abstract-op>Validate texture format required features</dfn> of a {{GPUTextureFormat}} |format|<br/>
```

Comment:
aside: Incidentally I also just noticed that our watermark text on timeline blocks disappeared, because we forgot to update the CSS to match the switch from `class=content-timeline` to `data-timeline=content`. That will cause a bunch of overlapping text, I'm sure. It will also cause the watermarks for validusage and data-timeline to compete. (I think data-timeline wins currently because it's specified later?)

Reply:
Ah, I was wondering if I had imagined that we had those watermarks at one point. Earmarking this as another follow up because I don't want to deal with checking all the potential overlaps on this one. :)